lms/templates/instructor/instructor_dashboard_2/e-commerce.html

```diff
@@ -16,6 +17,9 @@ <h3 class="error-msgs" id="error-msg"></h3>
             <span class="code_tip">
                 <p>${_('Create one or more pre-paid course enrollment codes. Students can use these codes to enroll in the course.')}</p>
                 <a id="registration_code_generation_link" href="#reg_code_generation_modal" class="add blue-button">${_('Create Enrollment Codes')}</a>
+
+                <p>${_('Invalidate, Re-Validate or Redeem a particular Registration code.')}</p>
```

Comment:
I think it'd be a bit clearer if it said "... or unredeemed a particular" a particular Registration code. I'll change it in a minute.